p2panda-js/test/session.test.ts

```diff
@@ -0,0 +1,46 @@
+import { expect } from 'chai';
+
+import { Session } from '../lib';
+
+// const PRIVATE_KEY =
+//   '4c21b14046f284f87f1ea4be4b973664221ad483079a68ed35a6812553b41176';
+
+describe('Session', () => {
+  it('can query entries', async () => {
+    const session = new Session('http://localhost:2020');
+    const entries = await session.queryEntries(
+      '00401d76566758a5b6bfc561f1c936d8fc86b5b42ea22ab1dabf40d249d27dd906401fde147e53f44c103dd02a254916be113e51de1077a946a3a0c1272b9b348437',
+    );
+    expect(entries.length).to.equal(2);
+  });
+
+  it('throws when querying without a schema', async () => {
+    const session = new Session('http://localhost:2020');
+    let error;
+    try {
+      await session.queryEntries();
+    } catch (e) {
+      error = e;
+    }
+    expect(error.message).to.equal('Schema must be provided');
+  });
+
+  it('gets next entry args', async () => {
+    const session = new Session('http://localhost:2020');
+    const { KeyPair } = await session.loadWasm();
+    const keyPair = new KeyPair();
+    const nextEntryArgs = await session.getNextEntryArgs(
+      keyPair.publicKey(),
+      '00401d76566758a5b6bfc561f1c936d8fc86b5b42ea22ab1dabf40d249d27dd906401fde147e53f44c103dd02a254916be113e51de1077a946a3a0c1272b9b348437',
+    );
+    expect(nextEntryArgs.entryHashSkiplink).to.equal('SKIPLINK_HASH');
```

Comment:
@cafca help! I've been staring at our openRPC definition file for so long and can't see anything wrong, but this test isn't getting the correct example values back from the mock server..... Can you see anything wrong with the `panda_getEntryArguments` definition? It seems to be validating correctly in the openRPC playground site, but obviously something isn't happy.

Reply:
I had a go at this yesterday. I didn't find any obvious errors as well. My approach now would be to construct a much simpler example and increase the complexity of the mock server until this error reappears.

Reply:
Thanks for checking! I found the problem now :sob:

Reply:
I didn't realise that when defining example responses, it checks against the example parameters you set. So you could have several different request and response examples (sounds so obious now I write it down, but I didn't realise it was doing this...). In our case it means we have to pass in the exact public_key string, rather than generating a new one on the fly....

Reply:
Oh, I just realised that you made changes to this branch. It has been a long time. I should check those commits of yours also. Actually, the process is a bit complex now because how can I review your changes when this is my own pr? hmm..

Reply:
Actually, I think it's all good :) Glad you were able to fix it!